README.rst

```diff
@@ -1,12 +1,11 @@
-HTTPie: human-friendly CLI HTTP client for the API era
+HTTPie: Human-friendly CLI HTTP client for the API era
 ######################################################
 
 HTTPie (pronounced *aitch-tee-tee-pie*) is a command-line HTTP client.
-Its goal is to make CLI interaction with web services as human-friendly as possible.
-HTTPie is designed for testing, debugging, and generally interacting with APIs & HTTP servers.
-The ``http`` & ``https`` commands allow for creating and sending arbitrary HTTP requests.
-They use simple and natural syntax and provide formatted and colorized output.
-
+Its goal is to make command-line interactions with web services as
+human-friendly as possible. The ``http`` & ``https`` commands create and
+send HTTP/S requests using a simple and natural syntax. Response data is
```

Comment:
maybe leave as HTTP instead of HTTP/S? since that's the usual way to refer to them

Reply:
I performed all the suggested edits in a new PR in a doc_update2 branch off this doc_update branch. Sorry for the confusion and multiple branches.

Reply:
@calwas please close this PR once you're ready

Reply:
@calwas please can you keep everything within this PR? There are quite a few specific suggestions here. It's important that we don't lose track of them, and make sure they have all been discussed and/or resolved.

Reply:
@elenalape Okay, I was able to update the original PR. All suggested edits were performed in this second commit.

Reply:
@elenalape @jakubroztocil I performed the suggested edits and submitted them three months ago. Are there additional edits still required before this PR can be committed? Cheers.